envergo/moulinette/models.py

```diff
@@ -1180,6 +1180,21 @@ def all_required_actions_interdit(self):
             for required_action in regulation.required_actions_interdit():
                 yield required_action
 
+    @property
+    def map(self):
+        """Returns a map to be displayed
+
+        Returns a `envergo.moulinette.regulations.Map` object.
+        This map object will be serialized to Json and passed to a Leaflet
+        configuration script.
+        """
+        return Map(
+            type="criterion",
+            center=self.catalog["coords"],
+            fixed=False,
+            entries=[],
+        )
```

Comment:
I reuse the _map_snippet.html template to display a map in the simulation sumary on print.
It is fed by this property, but the object envergo.moulinette.regulations.Map does not seems very well suited for this.
What do you think ?